### PR TITLE
Document the Chrome Web Store review process

### PR DESCRIPTION
**Pull request by dotproto**
This PR introduces a new page that documents how the Chrome Web Store review process works.

[Chrome Web Store review process](https://deploy-preview-1831--developer-chrome-com.netlify.app/docs/webstore/review-process/) staging site

### CODE REVIEW

File: site/en/docs/webstore/review-process/index.md [hunk 1]
```diff
@@ -0,0 +1,362 @@
...
+or more policy violations of moderate or greater severity in the published version of an extension.
+Second, delayed takedowns occur after the [warning period][warning] for a minor policy violation
+expires. In both cases, the impact of the takedown is the same.
+
+Developer communication
+
+: The publisher email address associated with the extension will receive an email stating that the
+  extension has been taken down due to one or more policy violations. In the case of an expired
+  warning, the email will include a reference to the warning email the developer should have
+  received.
```
Comment (jpmedley) [inline site/en/docs/webstore/review-process/index.md:239]:
```suggestion
  warning, the email will include a reference to the warning email the developer previously
  received.
```

File: site/en/docs/webstore/review-process/index.md [hunk 2]
```diff
@@ -0,0 +1,362 @@
...
+
+Chrome Web Store listing
+
+: When an extension is taken down, it will no longer be available in the Chrome Web Store. If normal
+  users attempt to access the extension's item listing, Chrome Web Store will return a 404 error. If
```
Comment (jpmedley) [inline site/en/docs/webstore/review-process/index.md:244]:
```suggestion
  users attempt to access the extension's listing, Chrome Web Store will return a 404 error. If
```

File: site/en/docs/webstore/review-process/index.md [hunk 3]
```diff
@@ -0,0 +1,362 @@
...
+  category listings, or elsewhere in the Chrome Web Store's consumer UI.
+
+Chrome UI
+
+: End users are not notified of the enforcement action immediately after takedown. If the violation
+  remains unresolved for several weeks, Chrome will automatically disable the extension and notify
+  the end user that the extension violates Chrome Web Store policy. Users may choose to re-enable
+  the extension if they wish.
+
+### Malware & extreme violations {: #malware }
```
Comment (jpmedley) [inline site/en/docs/webstore/review-process/index.md:258]:
```suggestion
### Malware and extreme violations {: #malware }
```

File: site/en/docs/webstore/review-process/index.md [hunk 4]
```diff
@@ -0,0 +1,362 @@
...
+
+The Chrome Web Store Review team has special procedures for egregious policy violations. In cases
+such as malware distribution, deceptive behavior designed to evade review, repeated severe
+violations indicative of malicious intent, and other egregious policy violations, more drastic
+measures are necessary.
+
+In order to limit the potential for these developers to further harm users, the Chrome Web Store
```
Comment (jpmedley) [inline site/en/docs/webstore/review-process/index.md:265]:
```suggestion
To limit the potential for these developers to further harm users, the Chrome Web Store
```

File: site/en/docs/webstore/review-process/index.md [hunk 5]
```diff
@@ -0,0 +1,362 @@
...
+  is suspended, the developer will receive notification of that enforcement action.
+
+Chrome Web Store listing
+
+: Just as with a [takedown][header-takedown], the offending item is removed from the Chrome Web
+  Store.
+
+Chrome UI
+
+: The violating extension is disabled on all end-user devices. Unlike standard takedowns, these
```
Comment (jpmedley) [inline site/en/docs/webstore/review-process/index.md:282]:
```suggestion
: The violating extension is disabled on all end user devices. Unlike standard takedowns, these
```

File: site/en/docs/webstore/review-process/index.md [hunk 6]
```diff
@@ -0,0 +1,362 @@
...
+
+{% Details %}
+{% DetailsSummary %}
+
+Appealing a review verdict
+
+{% endDetailsSummary %}
+
+1. Open the [One Stop Support contact form][page-oss-form].
+2. Select "My item (extensions, app, or theme)"
```
Comment (jpmedley) [inline site/en/docs/webstore/review-process/index.md:315]:
```suggestion
2. Select "My item (extensions, app, or theme)".
```

File: site/en/docs/webstore/review-process/index.md [hunk 7]
```diff
@@ -0,0 +1,362 @@
...
+3. Select "My item was warned / removed / rejected"
```
Comment (jpmedley) [inline site/en/docs/webstore/review-process/index.md:316]:
```suggestion
3. Select "My item was warned / removed / rejected".
```

File: site/en/docs/webstore/review-process/index.md [hunk 8]
```diff
@@ -0,0 +1,362 @@
...
+{% Details %}
+{% DetailsSummary %}
+
+Appealing an account suspension
+
+{% endDetailsSummary %}
+
+1. "My developer account"
+2. "My account was suspended".
+3. Provide additional details as requested by the form.
```
Comment (jpmedley) [inline site/en/docs/webstore/review-process/index.md:332]: Are these instructions or this this a list. If it's instructions, it needs verbs. If it's a list it needs to be bullets, not numbers. If the latter, it needs an intro explaining what this is a list of.

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: site/en/docs/extensions/whatsnew/index.md [hunk 9]
```diff
@@ -22,6 +22,11 @@ and related policy or other changes.
 now supports returning promises. When a script evaluates to a promise, Chrome will wait for the
 promise to settle and return its resulting value.
 
+### 2021.12.10: Chrome Web Store review documentation {: #cws-review-doc }
+
+Added [a new reference page](/docs/webstore/review-process) that provides an overview of the Chrome
+Web Store review process and explains how enforcement is handled.
```
[the remark this answers is not in the document]
Comment (dotproto) [reply site/en/docs/extensions/whatsnew/index.md:28]: Ack.

[the remark this answers is not in the document]
Comment (dotproto) [reply site/en/docs/extensions/whatsnew/index.md:28]: Ack. Revised text reads, "Added a new reference page that provides an overview of the Chrome Web Store review process and explains how Developer Program Policy enforcement is handled."

File: site/en/docs/webstore/review-process/index.md [hunk 10]
```diff
@@ -0,0 +1,362 @@
...
+{% Aside %}
+
+Existing published items may also be checked for these same violation; that process is described in
+the [Periodic review outcomes][header-periodic-outcomes] section.
+
+{% endAside %}
+
+Publish review requests have two basic outcomes.
+
+- **No violations are found** – The submission is approved and can be published to the Chrome Web
```
[the remark this answers is not in the document]
Comment (dotproto) [reply site/en/docs/webstore/review-process/index.md:104]: I'll use colons here.

File: site/en/docs/webstore/review-process/index.md [hunk 11]
```diff
@@ -0,0 +1,362 @@
...
+  Store.
+- **A violation is found** – The submission is [rejected][header-rejection] and the developer is
+  informed why.
+
+See [Developer communication][header-comms] for details on how these outcomes are communicated back
+to the developer and how developers can contact review regarding the outcome.
```
[the remark this answers is not in the document]
Comment (dotproto) [reply site/en/docs/webstore/review-process/index.md:110]: I'm not sure how we ended up with this wording, but I definitely agree that it's weird. I'll change this to this to "can contact the review team."

File: site/en/docs/webstore/review-process/index.md [hunk 12]
```diff
@@ -0,0 +1,362 @@
...
+Finally, a third potential outcome is that the submission is found to contain malware or another
+extreme policy violation. See [malware][header-malware] details on how these verdicts are enforced.
+
+### Periodic review outcomes {: #periodic-outcomes }
+
+This section describes how policy violations are handled during the periodic review process. Note
+that a violation identified during the [publishing review][header-publish-outcomes] process may
+trigger a re-review of the currently published version of the extension.
+
+{% Img src="image/SHhb2PDKzXTggPGAYpv8JgR81pX2/J0afA0FTf3TbbcOawOD3.png", alt="ALT_TEXT_HERE",
```
[the remark this answers is not in the document]
Comment (dotproto) [reply site/en/docs/webstore/review-process/index.md:121]: Adding alt text locally. Current alt text reads "Illustration of potential review outcomes and policy enforcement practices. Text details found in the 'Periodic review outcomes' and 'Violation enforcement' sections."

File: site/en/docs/webstore/review-process/index.md [hunk 13]
```diff
@@ -0,0 +1,362 @@
...
+### Takedown {: #takedown }
+
+Takedown refers to the act of removing an extension from the Chrome Web Store. In most situations
+takedowns are not permanent: the extension's developer can return the extension to the web store by
+submitting a new version that resolves the policy violation and passing the review process.
+
+Takedowns occur in two primary scenarios. First, immediate takedowns occur when reviewers detect one
+or more policy violations of moderate or greater severity in the published version of an extension.
+Second, delayed takedowns occur after the [warning period][warning] for a minor policy violation
+expires. In both cases, the impact of the takedown is the same.
```
[the remark this answers is not in the document]
Comment (dotproto) [reply site/en/docs/webstore/review-process/index.md:232]: I was trying to say that both immediate takedown and takedown after a warning are enforced the same way. I think this was mostly an attempt to head off potential developer support contacts where developers ask about differences in enforcement practices due to how the takdown was triggered.

File: site/en/docs/webstore/review-process/index.md (hunk 8 above)
Comment (dotproto) [reply site/en/docs/webstore/review-process/index.md:332]: This was meant as instructions. I don't recall writing these, so I *think* they may have been added by Andy.

In any case, I'm updating this list to use the same structure as the previous one and adding an intro sentence that sets context.

**Review by jpmedley — APPROVED** [review]

**Pull request merged**